src/core/Akka.FSharp/FsApi.fs

```diff
@@ -604,8 +610,20 @@ module Scheduler =
     /// <param name="message">Message to be sent to the receiver by the scheduler.</param>
     /// <param name="receiver">Message receiver.</param>
     /// <param name="scheduler"></param>
-    let scheduleTell (after: TimeSpan) (every: TimeSpan) (message: 'Message) (receiver: ActorRef) (scheduler: Scheduler): Async<unit> =
-        Async.AwaitTask (scheduler.Schedule(after, every, receiver, message).ContinueWith taskContinuation)
+    let scheduleTell (after: TimeSpan) (every: TimeSpan) (message: 'Message) (receiver: ActorRef) (scheduler: IScheduler): unit =
+         DeprecatedSchedulerExtensions.Schedule(scheduler, after, every, receiver, message)
```

Comment:
Just so I get this, what does this one do that requires it to use deprecated scheduler mechanics?